pypy/bin/checkmodule.py

```diff
@@ -14,21 +15,21 @@
 
 def main(argv):
     if len(argv) != 2:
-        print >> sys.stderr, __doc__
+        print( >> sys.stderr, __doc__)
```

Comment:
This runs? It looks really wrong to me. I think it (and the others) should be
```suggestion
        print(__doc__, file=sys.stderr)
```